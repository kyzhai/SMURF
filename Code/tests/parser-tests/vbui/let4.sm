
x = let y = 4 & z = 4 & v = 16 
    in y + z + v
