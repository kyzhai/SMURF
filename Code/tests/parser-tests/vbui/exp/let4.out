Program: 
	SVardef: 
		ID: x :: Unknown

	let  NEED TO FIX THIS PART  in y + z + v
Global Scope: 
	ID: x :: Unknown
	ID: print :: Unknown
	ID: random :: Unknown

